src/command_modules/azure-cli-managementpartner/azure/cli/command_modules/managementpartner/_help.py

```diff
@@ -0,0 +1,51 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['managementpartner'] = """
+    type: group
+    short-summary: Manage Azure ManagementPartner.
+"""
+
+helps['managementpartner create'] = """
+    type: command
+    short-summary: Create new management partner
+    long-summary: Add a management partner id to the current user
+    parameters:
+        - name: --partner-id
+          type: string
+          short-summary: Id of management partner
+"""
+
+helps['managementpartner get'] = """
+    type: command
+    short-summary: Get the management partner
+    long-summary: Get the management partner id for the current user
```

Comment:
this summary is not accurate as it requests to have id as input instead of return it. Based on the context, i would suppose it means `get management partner information`?

Reply:
@yugangw-msft , Do we have to use public prod to do the test? The service will be online tomorrow.

Reply:
You don't have to. I wanted to ensure your tests will not fail in our live run which runs in public env